vllm/envs.py

```diff
@@ -71,6 +71,7 @@
     VLLM_USE_V1: bool = False
     VLLM_ENABLE_V1_MULTIPROCESSING: bool = False
     VLLM_LOG_BATCHSIZE_INTERVAL: float = -1
+    VLLM_DISABLE_COMPILE_CACHE: bool = False
```

Comment:
added the flag to disable the compile cache. the compile cache is used by default.